html/captive-portal/templates/ssl_inspection/osx.html

```diff
@@ -0,0 +1,184 @@
+<!--Body container that sets layout and inner content-->
+    
+
+<!--Top navigation menu bar-->
+
+                <nav>
+                    <ul class="o-list-inline o-layout--center caption">
+                       <li class="o-list-inline__item u-padding-horizontal-tiny"><a href="/ssl_inspection/ios">iOS</a></li>
+
+                       <li class="o-list-inline__item u-padding-horizontal-tiny"><a href="/ssl_inspection/android">Android</a></li>
+                       
+                       <li class="o-list-inline__item u-padding-horizontal-tiny"><a href="/ssl_inspection/windows">Windows</a></li>
+                       
+                       <li class="o-list-inline__item u-padding-horizontal-tiny"><a href="/ssl_inspection/chrome">Chrome OS</a></li>
+                       
+                       <li class="o-list-inline__item u-padding-horizontal-tiny"><a href="/ssl_inspection/firefox">Firefox</a></li>
+                       
+                       <li class="o-list-inline__item u-padding-horizontal-tiny"><a href="/ssl_inspection/opera">Opera</a></li>
+                       
+                       <li class="o-list-inline__item u-padding-horizontal-tiny"><a href="/ssl_inspection/unsupported">Unsupported</a></li>
+		    </ul>
+                </nav>
+                    
+                    
+                <div id="guide">
+                    
+                        <p>
+                            [% i18n("This network employs a technology called captive portal which allows the administrator(s) to restrict network access to trusted users. In order to prevent certificate security warnings when accessing this portal, the network's filtering unit must issue to your computer or mobile device a file called an SSL certificate.") %]
+                        </p>
+
+                        <p>
+                       	    [% i18n("<strong>This guide will help you install the SSL filtering certificate for Mac-based browsers which use the operating system's built-in Keychain facility — this includes Safari and Chrome. This guide is not primarily for district owned Mac OSX computers.</strong>") | none %]
```

Comment:
.. Mac-based browsers which use the operating system's built-in Keychain facility => .. *browsers which use macOS built-in Keychain facility*
Mac OSX => *macOS*